output/install.nsi

```diff
@@ -207,11 +207,18 @@ program_files:
     File /nonfatal "weaseltARM.ime"
     File /nonfatal "weaseltARM64.ime"
   ${EndIf}
-  File "WeaselDeployer.exe"
-  File "WeaselServer.exe"
+  ${If} ${RunningX64}
```

Comment:
Maybe check for IsNativeAMD64 instead of RunningX64. RunningX64 will match ARM64 and therefore it will break Windows 10 on ARM (since x64 emulation is not supported there).

I can try to build ARM64 WeaselServer after this PR, winsparkle seems to support ARM64 as well so we should be OK.

Reply:
you mean the 64 bit WeaselServer.exe will not work well in WOA?

Reply:
Well 64-bit x86 emulation is not supported until Windows 11. We can do native ARM 64-bit, or keep it 32-bit, or maybe drop support for Windows 10 on ARM?

Reply:
I will make a new branch with `IsNativeAMD64 ` for testing, maybe after half an hour your can download it and try~

Reply:
opencc requires running the binaries (opencc_dict.exe) it just compiled to generate some data, which will be problematic for CI (we can't run ARM binaries on CI). Let's keep it 32-bit for now.

Reply:
> opencc requires running the binaries (opencc_dict.exe) it just compiled to generate some data, which will be problematic for CI (we can't run ARM binaries on CI). Let's keep it 32-bit for now.

opencc is running when building librime, x64 and Win32, and it's cached.

Reply:
Yeah if we want to build ARM librime it will be a problem.